notebooks/en/fine_tuning_llm_for_generate_persian_product_catalogs_in_json_format.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>maybe move this up ⬆️ where you're defining LoRA hyperparameters. </p><p>also for LoRA I think it's worth mentioning you are storing lower rank matrices that store changes in weights since it's the main point.</p><p>If you want you can always provide further reading at the bottom of the recipe not to divert the attention.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>